plugins/RocketsPlugin/RocketsPlugin.cpp

```diff
@@ -47,36 +47,47 @@
 
 namespace
 {
-const std::string ENDPOINT_API_VERSION = "v1/";
+// REST PUT & GET, JSONRPC set-* notification, JSONRPC get-* request
 const std::string ENDPOINT_ANIMATION_PARAMS = "animation-parameters";
 const std::string ENDPOINT_APP_PARAMS = "application-parameters";
 const std::string ENDPOINT_CAMERA = "camera";
-const std::string ENDPOINT_FRAME_BUFFERS = "frame-buffers";
 const std::string ENDPOINT_GEOMETRY_PARAMS = "geometry-parameters";
-const std::string ENDPOINT_IMAGE_JPEG = "image-jpeg";
 const std::string ENDPOINT_TRANSFER_FUNCTION = "transfer-function";
 const std::string ENDPOINT_RENDERING_PARAMS = "rendering-parameters";
 const std::string ENDPOINT_SCENE = "scene";
 const std::string ENDPOINT_SCENE_PARAMS = "scene-parameters";
-const std::string ENDPOINT_SIMULATION_HISTOGRAM = "simulation-histogram";
-const std::string ENDPOINT_STATISTICS = "statistics";
 const std::string ENDPOINT_STREAM = "stream";
-const std::string ENDPOINT_STREAM_TO = "stream-to";
+const std::string ENDPOINT_VOLUME_PARAMS = "volume-parameters";
+
+// REST GET, JSONRPC get-* request
+const std::string ENDPOINT_STATISTICS = "statistics";
 const std::string ENDPOINT_VERSION = "version";
+
+// REST GET
+const std::string ENDPOINT_FRAME_BUFFERS = "frame-buffers";
+const std::string ENDPOINT_SIMULATION_HISTOGRAM = "simulation-histogram";
 const std::string ENDPOINT_VOLUME_HISTOGRAM = "volume-histogram";
-const std::string ENDPOINT_VOLUME_PARAMS = "volume-parameters";
 
+// JSONRPC async requests
 const std::string METHOD_ADD_MODEL = "add-model";
-const std::string METHOD_CHUNK = "chunk";
+const std::string METHOD_SNAPSHOT = "snapshot";
+// METHOD_REQUEST_MODEL_UPLOAD from BinaryRequests.h
+
+// JSONRPC synchronous requests
 const std::string METHOD_GET_INSTANCES = "get-instances";
+const std::string METHOD_IMAGE_JPEG = "image-jpeg";
 const std::string METHOD_INSPECT = "inspect";
-const std::string METHOD_QUIT = "quit";
 const std::string METHOD_REMOVE_MODEL = "remove-model";
```

Comment:
Aren't `'remove-model'` and `'update-model'` jsonrpc notifications?

Reply:
You can use them with notifications if you want. But they can return true/false on success/error.